kong.go

```diff
@@ -249,8 +249,11 @@ func (k *Kong) interpolateValue(value *Value, vars Vars) (err error) {
 		vars = vars.CloneWith(varsContributor.Vars(value))
 	}
 
-	if value.Enum, err = interpolate(value.Enum, vars, nil); err != nil {
-		return fmt.Errorf("enum for %s: %s", value.Summary(), err)
```

Comment:
This Enum interpolation was duplicated and run twice. I removed it. This is unrelated to this PR though.